RELEASE_NOTES.md

```diff
@@ -7,6 +7,39 @@ in the release. For a detailed list of changed components, refer to
 This document contains release notes for the current major release and all patches.
 For prior releases, see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 12.10.0 (2016-10-31)
+
+### Chef Server
+- Smaller download - the download size has been reduced by around 35% via removal of redundant, cached, and unused components. The installed size has been similarly reduced.
+- add retry support to opscode-expander
+- `chef-server-ctl reindex` will now continue even if some objects are
+  not indexable, and will show which objects failed at the conclusion
+  of the run.
+- Data Collector support for Policyfiles.
+- `chef-server-ctl install` add-on installation now pulls from the
+  correct source.
+- Regression fix: that caused errors on reconifgure when LDAP bind password
+  is nil has been fixed.
+
+### Security Updates
+
+* Upgrade to OpenSSL 1.0.2j.  The prior release (1.0.1u) is approaching EOL.
+* Updated TLS ciphers. See compatibility notes, below.
+
+### Comaptibility Notes
+
+1. The change of TLS ciphers can cause older tooling to fail to negotiate
+   SSL sessions with the Chef Server. The changes to the cipher list are
+   captured [here](https://github.com/chef/chef-server/pull/918#issuecomment-244430458).
+   Upgrading any custom clients of the Chef Server API to use a current SSL
+   release will resolve this.
+2. With this TLS cipher suite change, the Reporting add-on will report
```

Comment:
I think you already helped me do that - https://github.com/chef/oc_reporting/commit/7c5a2b27d2f81e3e3b0bd0fd2ee5bc3258d1e13a